docs/content/preview/releases/ybdb-releases/v2.20.md

```diff
@@ -30,6 +30,10 @@ What follows are the release notes for the YugabyteDB v2.20 release series. Cont
 
 For an RSS feed of all release series to track the latest product updates, point your feed reader to the [RSS feed for releases](../index.xml).
 
+{{<warning title="Changes to supported operating systems">}}
+YugabyteDB 2.20.3.0 and newer releases supports Red Hat Enterprise Linux 9.3 on x86-based systems. Refer to [Operating system support](../../../reference/configuration/operating-systems/) for the complete list of supported operating systems.
```

Comment:
```suggestion
YugabyteDB 2.20.3.0 and later releases support Red Hat Enterprise Linux 9.3 on x86-based systems. Refer to [Operating system support](../../../reference/configuration/operating-systems/) for the complete list of supported operating systems.
```

Reply:
Seems like a New feature - "Added support for Red Red Hat Enterprise Linux 9.3 on x86-based systems. Refer to [Operating system support](../../../reference/configuration/operating-systems/) for the complete list of supported operating systems."